botocore/signers.py

```diff
@@ -586,6 +587,8 @@ def generate_presigned_url(self, ClientMethod, Params=None, ExpiresIn=3600,
     prepare_request_dict(
         request_dict, endpoint_url=self.meta.endpoint_url, context=context)
 
+    request_dict['context']['partition'] = self.meta.partition
```

Comment:
Crap, I meant to take this out.  We don't need `partition` in the request context.